source/documentation/deploying_apps/production_checklist.md

```diff
@@ -5,3 +5,4 @@ Before deploying an app for production use, check the following:
 1. If your app uses a PostgreSQL service, make sure that you have selected the high-availability plan (``M-HA-dedicated-9.5``) for the bound ``postgres`` service instance.
 2. You are running at least two instances of the app to ensure availability. Use ``cf scale APPNAME -i 2`` to add an extra running instance.
 3. You are prepared to use a [blue-green deployment process](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) [external link] for when the app needs to be updated or restarted (this avoids problems due to a known issue with the PaaS that can generate transient 404 errors).
+4. You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) [external link] for your app.
```

Comment:
Thanks for the contribution.

The default `health-check-type` is `port`. Should this advice be more explicit in saying that you should use `http` and ensuring that `health-check-http-endpoint` is correct for your app?